### PR TITLE
LPD-17542_2 "Home" folder was renamed to "site name" folder name

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:document

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2539649158

File: portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro [hunk 1]
```diff
@@ -2356,16 +2356,22 @@ definition {
 
 		SelectFrame(locator1 = "IFrame#MODAL_BODY");
 
-		if (${dmFolderName} == "Home") {
-			if (IsElementPresent(key_breadcrumbName = "Home", locator1 = "BreadcrumbPortlet#BREADCRUMB_FOLDER")) {
+		echo("## * dmFolderName: ${dmFolderName}");
```
Comment (jkappler) [inline portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro:2359]: @AliciaGarciaGarcia, shouldn't we remove the echos from the tests?

Comment (austinchiang) [reply portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro:2359]: This could be helpful when debugging a test failure

**Review by austinchiang — APPROVED** [review]

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward:force

**Pull request closed without merging**